libbeat/outputs/elasticsearch/enc.go

```diff
@@ -94,6 +95,8 @@ func (b *jsonEncoder) AddRaw(obj interface{}) error {
 		err = b.folder.Fold(event{Timestamp: v.Timestamp, Fields: v.Fields})
 	case *beat.Event:
 		err = b.folder.Fold(event{Timestamp: v.Timestamp, Fields: v.Fields})
+	case report.Event:
+		err = b.folder.Fold(report.Event{Timestamp: v.Timestamp, Fields: v.Fields})
```

Comment:
this should not be necessary. You unpack and pack the event... the default clause should have done the trick

Reply:
I wish there was a unisex facepalm emoji in github. I would use it now. :D